recipes/install.rb

```diff
@@ -32,5 +36,5 @@
 end
 
 service 'grafana-server' do
-  action [:enable, :start]
+  action :enable
```

Comment:
This change looks like the service isn't started so would impact folks. Can you give more information about why the change here?

Reply:
With `:start` service was started before `configuration` recipe kicked-in. And admin-password was set to the default one. [It should not be a problem because change to grafana.ini will induce restart.](https://github.com/chef-cookbooks/chef-grafana/blob/master/recipes/configure.rb#L6).